build.sbt

```diff
@@ -1,10 +1,14 @@
+import sbt.Keys.publishLocalConfiguration
+
 ThisBuild / resolvers += Resolver.mavenLocal
 ThisBuild / scalaVersion     := "2.12.15"
-ThisBuild / version          := "0.7.2"
+ThisBuild / version          := "0.7.3"
```

Comment:
Should we bump it now? Maybe we should do it before the release

Reply:
Will defer